src/server/raft_client.rs

```diff
@@ -892,6 +892,9 @@ where
                         if !s.full {
                             s.full = true;
                             self.full_stores.push((store_id, conn_id));
+                            CHANNEL_FULL_COUNTER_VEC
```

Comment:
It should be recorded in `kv.rs`, this is a different case.

Reply:
Thanks,
I have now moved this to the `handle_raft_message` function in `kv.rs`.